n2y/blocks.py

```diff
@@ -158,7 +158,10 @@ def to_pandoc(self):
         content = [Plain(self.rich_text.to_pandoc())]
         if self.has_children:
             children = self.children_to_pandoc()
-            content.extend(children)
```

Comment:
Try wrapping in a paragraph.

Also, add a detailed comment explaining why we needed to do this.

Reply:
Paragraph wrap didn't work